api/admin/controller.py

```diff
@@ -1051,6 +1051,8 @@ def libraries(self):
             library.short_name = short_name
 
         for setting in Configuration.LIBRARY_SETTINGS:
+            # Start off by assuming the value is not set.
+            value = ""
```

Comment:
I'm not sure how to test this. The problem I found was that when I didn't upload a logo, the value of the 'logo' configuration option was set to the value I'd set for "Color scheme", the previous configuration option.

Reply:
This code might introduce another problem I've noticed: if I upload a logo, and then make a change, the logo disappears.

Reply:
Good catch. Maybe the value should be None instead of empty string. You could change tests/admin/test_controller.py:TestSettingsController.test_libraries_post_edit to test this but it depends on the order of the settings so it'd be hard to cover everything.

Reply:
The second problem is interesting. Usually if you make a change and don't include things you previously set, that's interpreted as removing them. That doesn't work for images though.

Reply:
See if the change I made works for you.

Reply:
Yeah, that's fine, although we might want to add a way to delete your image without replacing it as well.